addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComHandler.java

```diff
@@ -45,14 +44,11 @@
  * @author Pauli Anttila - Initial contribution
  */
 public class RFXComHandler extends BaseThingHandler implements DeviceMessageListener {
+    private final static int LOW_BATTERY_LEVEL = 1;
 
-    private Logger logger = LoggerFactory.getLogger(RFXComHandler.class);
+    private static final Logger LOGGER = LoggerFactory.getLogger(RFXComHandler.class);
```

Comment:
was this actually a check of the analysis tool? If so, we need to adapt it - our coding guidelines state that non-static loggers should be used whenever possible.